### ISSUE TITLE
The loss was very low at the beginning

### ISSUE THREAD

**Issue by lim-0** [Holmeyoung/crnn-pytorch#44]
firstly,thank you for your code.
I have 20000 images for training and 2000 for testing(all generate by code)  when i started training the loss was always very small(e.g 0.09),can you tell me how can I deal with it.

**Comment by ghost** [comment]
how is the performance on the testing data? maybe training data size is too small, so that it is very easy for the network to fit?

**Comment by lim-0** [comment]
> how is the performance on the testing data? maybe training data size is too small, so that it is very easy for the network to fit?

thank you for your answer.The test result is good,as you say ,may be the data I generated is too esay to fit.
